x/observer/keeper/voting.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/zeta-chain/zetacore/x/observer/types"
 )
 
+const (
+	errVoteOnBallot = "error while voting on ballot"
```

Comment:
I think variable prefixed with err generally implied being of error type, don't we break this convention here?

Reply:
We can rename it to msgFooBar in that case, assuming the convention for err prefix would be for variables of type error.